summarization/summarizer_main.py

```diff
@@ -342,3 +351,17 @@ def _get_stamp_descriptor_for_embedded_content(self, content):
         # type casting is required since the dict keys
         # are now of type str
         return self.embedded_descriptors_dict[str(content.content_type)]
+
+    def _filter_images_with_text(self):
```

Comment:
This method also creates STAMP pages for media with overlay text. Rename accordingly.

Reply:
updated.